src/main/java/com/linkedin/kmf/services/ConsumeService.java

```diff
@@ -265,9 +262,9 @@ public ConsumeMetrics(Metrics metrics, final Map<String, String> tags) {
         new Measurable() {
           @Override
           public double measure(MetricConfig config, long now) {
-            double recordsConsumedRate = _sensors.metrics.metrics().get(new MetricName("records-consumed-rate", METRIC_GROUP_NAME, tags)).value();
-            double recordsLostRate = _sensors.metrics.metrics().get(new MetricName("records-lost-rate", METRIC_GROUP_NAME, tags)).value();
-            double recordsDelayedRate = _sensors.metrics.metrics().get(new MetricName("records-delayed-rate", METRIC_GROUP_NAME, tags)).value();
+            double recordsConsumedRate = metrics.metrics().get(metrics.metricName("records-consumed-rate", METRIC_GROUP_NAME, tags)).value();
```

Comment:
I don't know very much about this metrics API so I assume this is the correct thing to do.

Reply:
Yeah it is needed for Kafka 0.11.